processor/samplingprocessor/tailsamplingprocessor/idbatcher/id_batcher_test.go

```diff
@@ -89,9 +91,7 @@ func BenchmarkConcurrentEnqueue(b *testing.B) {
 
 func concurrencyTest(t *testing.T, numBatches, newBatchesInitialCapacity, batchChannelSize uint64) {
 	batcher, err := New(numBatches, newBatchesInitialCapacity, batchChannelSize)
-	if err != nil {
-		t.Fatalf("Failed to create Batcher: %v", err)
-	}
+	require.NoError(t, err, "Failed to create Batcher: %v", err)
```

Comment:
should this be `require.NoErrorf`

Reply:
Yes. I will fix it also in my other PRs.

Reply:
I discovered that `require.NoError` works like `require.NoErrorf`